.evergreen/run-csfle-tests.sh

```diff
@@ -26,7 +26,7 @@ CARGO_RESULT=0
 
 cargo_test() {
     RUST_BACKTRACE=1 \
-        cargo test --features ${FEATURE_FLAGS} $1 ${OPTIONS} | cargo2junit
+        cargo test --features ${FEATURE_FLAGS} $1 ${OPTIONS} | grep -v '{"t":' | cargo2junit
```

Comment:
The other thing I have no dang idea about - occasionally some output from the spawned `mongocryptd` shows up in the stdout of the test, causing `cargo2junit` to barf.  This shouldn't be happening - stdout and stderr are closed before spawning it, and I've validated that behavior with a test binary - but there it is.

Reply:
weird 🤷‍♀️ filtering out the test lines seems reasonable to me